.cm/change_categories.cm

```diff
@@ -197,7 +197,7 @@ platforms:
   - jvm:
     name: 'jvm'
     subprojects:
-      - 'subprojects/code-quality/'
+      - 'platforms/code-quality/'
```

Comment:
`platforms/code-quality/` does not exist. `platforms/jvm/code-quality/` does.
We should likely remove this line entirely and update the `platforms/jvm/ear/` line below to be just `platforms/jvm`